e2e/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "bracketSpacing": false,
+    "printWidth": 120,
+    "singleQuote": true,
+    "tabWidth": 4
+}
```

Comment:
0/5 should we use a global confirmation for all webapp node and not only the e2e code?

Reply:
Yup, sounds good.